.github/workflows/backup-and-restore.yml

```diff
@@ -0,0 +1,43 @@
+name: Restore Backup to AKS Databases
+#Restores AKS Databases from their PaaS equivalent
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: GitHub environment to backup and restore
+        type: choice
+        default: qa
+        options:
+          - qa
+          - staging
+
+jobs:
+  backup-and-restore:
+    name: ${{ github.event.inputs.environment }} backup and restore to aks
+    environment:
+      name: ${{ github.event.inputs.environment }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Backup and Restore action for ${{ inputs.environment }} to Test AKS cluster
+        if: inputs.environment == 'qa' || inputs.environment == 'staging'
+        id: backup_and_restore_test_env
+        uses: ./.github/actions/backup_and_restore/
+        with:
+          environment: ${{ github.event.inputs.environment }}
+          azure-credentials-paas: ${{ secrets.AZURE_CREDENTIALS }}
+          azure-credentials-aks: ${{ secrets.AZURE_CREDENTIALS_AKS_TEST }}
+          slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
+
+      - name: Backup and Restore action for ${{ inputs.environment }} to Prod AKS cluster
```

Comment:
Could we have only 1 backup and Restore step? We could have 2 seprate steps to set the azure-credentials-aks variable instead

Reply:
As discussed, this would add too much complexity as it's a sensitive value